client/components/Menu/Menu.tsx

```diff
@@ -73,8 +73,8 @@ export const Menu = React.forwardRef((props: MenuProps, ref) => {
 				as="ul"
 				style={{ zIndex: 20, ...menuStyle }}
 				className={classNames(Classes.MENU, Classes.ELEVATION_1, className)}
-				unstable_portal={menuConfig.usePortal}
 				{...menu}
+				modal={menuConfig.usePortal}
```

Comment:
It seems `unstable_portal` has been deprecated for declaring if things will render in modal